lib/rubycritic/commands/default.rb

```diff
@@ -12,7 +13,8 @@ def initialize(paths)
       end
 
       def execute
-        report(critique)
+        return report(critique) if Config.format == :json
```

Comment:
This will fail soon since a PR with a consolereport ist almost done.
Also I feel a 

```Ruby
case Config.format
when ....
end
```

is more expressive since it makes it clear that we are choosing paths based on format whereas your "return" makes it seem like json would be a special case.